src/app/tests/suites/certification/Test_TC_BRBINFO_2_1.yaml

```diff
@@ -140,21 +140,16 @@ tests:
       response:
           value: ProductNameValue
 
-    - label:
-          "Step 14: TH reads attribute ID 4 from the DUT (matches in ID to
-          ProductID in the parent cluster, but is absent on the
-          BridgedDeviceBasicInformation cluster)."
+    - label: "Step 14: TH reads ProducID from the DUT"
       PICS: BRBINFO.S
-      cluster: "AnyCommands"
-      command: "ReadById"
-      arguments:
-          values:
-              - name: "ClusterId"
-                value: BRBINFO.ClusterId
-              - name: "AttributeId"
-                value: 0x0004
+      command: "readAttribute"
+      attribute: "ProductID"
       response:
-          error: UNSUPPORTED_ATTRIBUTE
+          saveAs: ProductIDValue
```

Comment:
nit, no need to save if you don't use the value

Reply:
ack